gammapy/time/plot.py

```diff
@@ -30,3 +32,167 @@ def plot_time_difference_distribution(time, ax=None):
 
     # TODO: implement!
     raise NotImplementedError
+
+
+def plot_fermi_3fgl_light_curve(name_3fgl, time_start=None, time_end=None, ax=None):
+    """Plot flux as a function of time for a fermi 3FGL object.
+
+    Parameters
+    ----------
+    name_3FGL : `string`
+        The 3FGL catalog name of the object to plot
+    time_start : `~astropy.time.Time`
+        Light curve start time.  If none, use the earliest time in the catalog.
+    time_end : `~astropy.time.Time`
+        Light curve end time.  If none, use the latest time in the catalog.
+    ax : `~matplotlib.axes.Axes` or None
+        Axes
+
+    Returns
+    -------
+    ax : `~matplotlib.axes.Axes`
+        Axes
+
+    Examples
+    --------
+    Plot effective area vs. energy:
+
+    .. plot::
+        :include-source:
+
+        from astropy.time import Time
+        from gammapy.time import plot_fermi_3fgl_light_curve
+        import matplotlib.pyplot as plt
+
+        time_start = Time('2010-01-01T00:00:00')
+        time_end = Time('2015-02-02T02:02:02')
+
+        plt.plot = plot_fermi_3fgl_light_curve('3FGL J0349.9-2102', time_start, time_end)
```

Comment:
For me this example looks like this:

![image](https://cloud.githubusercontent.com/assets/852409/8277633/327f8298-18c5-11e5-9b1a-a22534d6f0f9.png)

The display of upper limits with the arrows pointing up is confusing, no?

They've probably discussed this at length ... maybe it's best to just plot it the way it's done in the 3FGL paper?
![screen shot 2015-06-22 at 09 59 48](https://cloud.githubusercontent.com/assets/852409/8277667/7370cfe6-18c5-11e5-8683-fb20ed66eb29.png)